SU-SWS/stanford_private_page
# Basic 1615

Adds a jQuery plugin that scans the whole page for private-page links and titles. If it finds a matching element it then overrides spoken output by adjusting the `aria-label` property on the matching element. 

For links it defaults to "[Link Text] Requires Login"
For page titles it defaults to "[Page title] Protected Page"

To Test:
1. Pull down this latest version on to a site and enable or pull down on to an existing site.
2. Create or find content that uses the `.private-link` css class
3. Enable voice over or something like chrome vox to test the spoken output. 
4. Edit the link with the .private-link css class and add or change the `title` attribute
5. The spoken output for the link with the new title should be [Link text][Title]

The title attribute is how the individual links will be overridden.
